packages/evm/src/params.ts

```diff
@@ -272,8 +272,9 @@ export const paramsEVM: ParamsDict = {
    */
   2935: {
     // evm
-    historyStorageAddress: '0x0aae40965e6800cd9b1f4b05ff21581047e3f91e', // The address where the historical blockhashes are stored
+    historyStorageAddress: '0xfffffffffffffffffffffffffffffffffffffffe', // The address where the historical blockhashes are stored
     historyServeWindow: 8192, // The amount of blocks to be served by the historical blockhash contract
+    systemAddress: '0xfffffffffffffffffffffffffffffffffffffffe', // The system address
```

Comment:
Do we need to have both of these parameters since they are identical?

Reply:
I will admit that I've been a bit confused with regards to those two addresses. I've played around when troubleshooting so this is a remnant of that. I've adjusted it so that it matches the EIP: https://eips.ethereum.org/EIPS/eip-2935